backend/routes/web.js

```diff
@@ -7,9 +7,15 @@ import authenticateToken from "../middleware/auth.js";
 const router = Router();
 
 // Get Requests
+<<<<<<< HEAD
+router.get('/usernames', UserController.getAllUserName)
+router.get('/token', authenticateToken, UserController.verifyUserByToken)
+router.get("/recipes", RecipeController.allRecipe)
+=======
 router.get("/usernames", UserController.getAllUserName);
 router.get("/token", authenticateToken, UserController.verifyUserByToken);
 router.get("/recipes", RecipeController.allRecipe);
+>>>>>>> 2c3a6840f60182553c9345c7f4a9af5057c8f6a0
 // added route to get previous comments
```

Comment:
@raj-adi00 the above <<<< Head like part will give syntactical error please remove it. This is due to not resolving the conflicts properly